eng/common/pipelines/templates/steps/prepare-pipelines.yml

```diff
@@ -135,3 +157,22 @@ steps:
       condition: ne('${{parameters.TestsConventionOptions}}','')
       env:
         PATVAR: $(azuresdk-azure-sdk-devops-pipeline-generation-pat)
+    - script: >
+        $(Pipeline.Workspace)/pipeline-generator/pipeline-generator
+        --organization https://dev.azure.com/azure-sdk
+        --project internal
+        --prefix ${{parameters.Prefix}}-pr
+        --devopspath "\${{parameters.Prefix}}\pr"
+        --path $(System.DefaultWorkingDirectory)/sdk
+        --endpoint Azure
+        --repository ${{parameters.Repository}}
+        --convention "tests-weekly"
+        --agentpool Hosted
+        --branch refs/heads/$(DefaultBranch)
+        --patvar PATVAR
+        --debug
+        ${{parameters.TestsWeeklyConventionOptions}}
```

Comment:
Is `${{parameters.TestsWeeklyConventionOptions}}` a parameter for the script or value of debug?
Could you follow the above format or make a little clear?

Reply:
I agree it can be slightly misleading. In this case, I'm following the convention laid out in the rest of the file (e.g. [here](https://github.com/Azure/azure-sdk-tools/blob/dadafb368e1b7aaa3b0ac534755530987eb3a19d/eng/common/pipelines/templates/steps/prepare-pipelines.yml#L57) which gets set/passed in for example [here](https://github.com/Azure/azure-sdk-for-go/blob/main/eng/pipelines/prepare-pipelines.yml#L10)).